charts/plgd-hub/values.yaml

```diff
@@ -64,13 +64,13 @@ nats:
   cluster:
     enabled: false
     noAdvertise: false
-
   leafnodes:
     enabled: false
     noAdvertise: false
-
   natsbox:
     enabled: false
+  reloader:
+    image: ghcr.io/plgd-dev/hub/nats-server-config-reloader:vnext-pr956
```

Comment:
Won't this image get deleted when you close this PR?

Reply:
Sure. I want to change it to vnext.